AmplifyTests/CategoryTests/DataStore/Model/ListTests.swift

```diff
@@ -165,131 +165,6 @@ class ListTests: XCTestCase {
         XCTAssertEqual(json, "[]")
     }
 
-    func testLoadWithCompletion() throws {
-        XCTAssertEqual(ModelListDecoderRegistry.listDecoders.get().count, 0)
-        let data: JSONValue = [
-            ["id": "1"],
-            ["id": "2"]
-        ]
-
-        let serializedData = try ListTests.encode(json: data)
-        let list = try ListTests.decode(serializedData, responseType: BasicModel.self)
-        guard case .notLoaded = list.loadedState else {
-            XCTFail("Should not be loaded")
-            return
-        }
-        let loadComplete = expectation(description: "Load completed")
-        list.load { result in
-            switch result {
-            case .success(let elements):
-                XCTAssertEqual(elements.count, 2)
-                loadComplete.fulfill()
-            case .failure(let dataStoreError):
-                XCTFail("\(dataStoreError)")
-            }
-        }
-        wait(for: [loadComplete], timeout: 1)
-        guard case .loaded = list.loadedState else {
-            XCTFail("Should be loaded")
-            return
-        }
-        let loadedStateLoadComplete = expectation(description: "Load when in Loaded state completed")
-        list.load { result in
-            switch result {
-            case .success(let elements):
-                XCTAssertEqual(elements.count, 2)
-                loadedStateLoadComplete.fulfill()
-            case .failure(let dataStoreError):
-                XCTFail("\(dataStoreError)")
-            }
-        }
-        wait(for: [loadedStateLoadComplete], timeout: 1)
-    }
-
-    func testLoadWithCompletionWhenUnderlyingDataStoreError_InternalOperationFailure() {
-        let mockListProvider = MockListProvider<BasicModel>(
-            elements: [BasicModel](),
-            error: .listOperation("", "", DataStoreError.internalOperation("", "", nil))).eraseToAnyModelListProvider()
-        let list = List(listProvider: mockListProvider)
-        let loadComplete = expectation(description: "Load completed")
-        list.load { result in
```

Comment:
i think this test file is missing `list.fetch` test, right?

Reply:
There are success and failure tests for fetch API here :
https://github.com/aws-amplify/amplify-ios/blob/dev-preview/AmplifyTests/CategoryTests/DataStore/Model/ListPaginationTests.swift